.github/workflows/gradle.yml

```diff
@@ -0,0 +1,17 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with Gradle
+      run: ./gradlew build -PmavenDataPipelineRepo=datapipeline -PmavenDataPipelineUrl=https://maven.northconcepts.com/repositories/datapipeline -PmavenDataPipelineGroupId=com.northconcepts -PmavenUser=${{ secrets.mavenUser }} -PmavenPassword=${{ secrets.mavenPassword }} --no-daemon
```

Comment:
@Bryan-Noel do we need these arguments to run the build?  If not, can you drop them?